.editorconfig

```diff
@@ -9,9 +9,13 @@ charset=utf-8
 
 [*.java]
 indent_style=tab
-max_line_length=120
 
 [*.kt]
 indent_style=space
 indent_size=4
+
+[*.{kt, java}]
 max_line_length=120
+ij_kotlin_name_count_to_use_star_import = 999
+ij_kotlin_name_count_to_use_star_import_for_members = 999
+ij_java_class_count_to_use_import_on_demand = 999
```

Comment:
there is not "disabled" option, like 0 or -1?

Reply:
Android Studio has the setting but I could not find that there is such a setting for editorconfig